MaterialDesignThemes.Wpf/PopupBox.cs

```diff
@@ -511,6 +571,15 @@ protected void Close()
         {
             if (IsPopupOpen)
                 SetCurrentValue(IsPopupOpenProperty, false);
+
+            if (_popupContentControl != null && !_popupContentControl.IsKeyboardFocusWithin)
+            {
+                supressfirstElementGotFocus = true;
+
+                if (firstchildelement != null)
+                    firstchildelement.GotKeyboardFocus -= InnerFirstElement_GotKeyboardFocus;
+                    firstchildelement.LostKeyboardFocus -= InnerFirstElement_LostKeyboardFocus;
```

Comment:
I would prefer
```
firstchildelement?.GotKeyboardFocus -= InnerFirstElement_GotKeyboardFocus;
firstchildelement?.LostKeyboardFocus -= InnerFirstElement_LostKeyboardFocus;
```
as the null checks simply serves to prevent exceptions.

Also the if statement only captures the first expression.

Reply:
Thanks for the tip @w-syss ! I've worked on vb.net for more than 10 years, and sometimes I'm moving to c# for less then 1 year. There's lot of thinks to learn yet.

Reply:
![image](https://user-images.githubusercontent.com/15158923/83949872-9d3f7400-a7fc-11ea-82bd-d925767759a4.png)

C# are hiting me a lot!

Error CS0079: The event "IInputElement.GotKeyboardFocus" can only appear on the left side of + = or - =

Reply:
Could I use the common "if (condition) { true code; }" here to resolve?

Reply:
My suggestion was wrong. The `?` operator can obviously not be used on the left hand side of an assignment :D

if would go with:

```
if (!(firstchildelement is null))
{
    ...
}
```

Reply:
Right. Let's resolve by this way. Thanks.